client/src/pages/automation/project/components/node-details-tabs/ConnectionTab.tsx

```diff
@@ -74,7 +74,10 @@ const ConnectionTab = ({
     return (
         <div className="h-full flex-[1_1_1px] overflow-auto p-4">
             {connections?.length ? (
-                <Select onValueChange={(value) => handleValueChange(+value)} value={connectionId + ''}>
+                <Select
+                    onValueChange={(value) => handleValueChange(+value)}
+                    value={connectionId ? connectionId + '' : undefined}
```

Comment:
zašto dodajemo prazan string na `connectionId` (connectionId + '').

ovako su se znali prije minjat tipovi podatka iz number u string, ali to ne želimo radit više, pogotovo ne uz typescript